KarhooUISDK/Extensions/UIColor+Extensions.swift

```diff
@@ -68,4 +68,10 @@ extension UIColor {
             return lightModeColor
         }
     }
+    
+    func isLight() -> Bool {
```

Comment:
Nice hacking. I'd change to read-only variable though, in order to align to Apple SDK common.

Reply:
I see it as necessary whenever having text on primary / secondary / accent colors. The DPs can customize the colors at a global level, so we may end up in situations where the UI is unusable because of their customizations. I encourage the team to check the brightness in this scenario. If the text is on normal background, then checking the brightness is not necessary.

Reply:
Made it a variable instead of a method